backend/src/services/testService.ts

```diff
@@ -0,0 +1,10 @@
+import { executeQuery } from '../db/queries';
+import { createUser } from './userService';
+
+/**
+ * Initializes the users for the tests
+ */
+export const initializeUsers = async () => {
+    await executeQuery('DELETE FROM users', []);
```

Comment:
The database should get wiped before each unit test. Is this query really necessary?

Reply:
I'll go delete that line 👍